packaging/windows/netdata.manifest.in

```diff
@@ -0,0 +1,24 @@
+{
+  "PackageIdentifier": "netdata.netdata",
+  "PackageVersion": "1.0.0",
+  "PackageName": "Netdata",
+  "Publisher": "Netdata Inc.",
+  "Description": "Netdata is a high-performance, cloud-native, and on-premises observability platform designed to monitor metrics and logs with unparalleled efficiency.",
+  "License": "GPL-3.0",
+  "LicenseUrl": "https://www.gnu.org/licenses/gpl-3.0.txt",
```

Comment:
This is blatantly wrong, it needs to point to a unified license that covers the whole package.